### DISCUSSION TITLE
Cloudflare Workers Frameworks

### DISCUSSION THREAD

**Discussion by r3dpoint** [remix-run/remix#10085] [Q&A]
Looking for clarification on which Remix template to use when using Cloudflare's newest offering Cloudflare Workers Frameworks https://developers.cloudflare.com/workers/frameworks/ 

- Cloudflare CLI will bring in Remix's Cloudflare Pages template
- Remix has both a Cloudflare Workers template (for a legacy Worker stack?) and a Cloudflare Pages template

From what I've gleaned Cloudflare Workers Frameworks is their strategic replacement to Cloudflare Pages (preview builds, full stack apps), but since it is on Workers features like Cloudflare Pages middleware no longer exist. It is this removal of Cloudlfare Pages features that raises which Remix template to use.

Or is Remix planning to create a 3rd template to support this new flavor of Cloudflare?

**Comment by gustavopch** [comment]
FWIW, there's an open PR to update the Cloudflare templates to use Static Assets: https://github.com/remix-run/remix/pull/10034
